src/graph/ConfigExecutor.cpp

```diff
@@ -144,20 +144,22 @@ void ConfigExecutor::setVariables() {
     auto future = ectx()->gflagsManager()->setConfig(module, name, type, value);
     auto *runner = ectx()->rctx()->runner();
 
-    auto cb = [this] (auto && resp) {
+    auto cb = [this, name] (auto && resp) {
```

Comment:
Please note that `name` here will be copied. But I think it's OK for this scenario.

Reply:
The name itself is allocated in stack, when this cb is called, the context has been changed, so it must be copied.